### PR TITLE
[17.0] [MIG] product_logistics_uom: Migration to 17.0 

### PR DESCRIPTION
**Pull request by bizzappdev**

### CODE REVIEW

File: product_logistics_uom/models/res_config_settings.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        "uom.uom",
+        "Default Volume Unit of Measure",
+        domain=lambda self: [
+            ("category_id", "=", self.env.ref("uom.product_uom_categ_vol").id)
+        ],
+        config_parameter="product_default_volume_uom_id",
+        help="Default unit of measure to express product volume",
+    )
+
+    product_default_length_uom_id = fields.Many2one(
```
Comment (nguyenminhchien) [inline product_logistics_uom/models/res_config_settings.py:31]: I don't see this field ( `product_default_length_uom_id` ) on the form view.

**Review by nguyenminhchien — CHANGES_REQUESTED** [review]
I have 1 comment, others look good.

**Comment by ferran-S73** [comment]
LGTM, same as already said if you could please add the product_default_length_uom_id field to the settings view

File: product_logistics_uom/models/res_config_settings.py (hunk 1 above)
Comment (bizzappdev) [reply product_logistics_uom/models/res_config_settings.py:31]: It was also not added to older versions. But I agree with you it should be visible.
We have added it in the view.

**Review by ferran-S73 — APPROVED** [review]

**Review by nguyenminhchien — APPROVED** [review]

File: product_logistics_uom/models/res_config_settings.py (hunk 1 above)
Comment (nguyenminhchien) [reply product_logistics_uom/models/res_config_settings.py:31]: thank you.

**Review by PCatinean — APPROVED** [review]

**Pull request merged**